data/src/main/java/org/oppia/android/data/backends/gae/api/PlatformParameterService.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.android.data.backends.gae.api
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/** Service that provides access to the Oppia platform parameter endpoint. */
+interface PlatformParameterService {
+
+  /**
+   * This function returns a Map of Platform Parameters where the keys corresponds to parameter
```

Comment:
```suggestion
   * Retrieves map of platform parameters where the keys corresponds to parameter
```

"This function" is implied for the summary fragment and can be omitted (in the same way that "This class" is omitted from class-level KDoc summary fragments).

Reply:
Done